ghcide/test/exe/CradleTests.hs

```diff
@@ -210,7 +210,7 @@ sessionDepsArePickedUp = testWithDummyPluginEmpty'
         "cradle: {direct: {arguments: []}}"
     -- Open without OverloadedStrings and expect an error.
     doc <- createDoc "Foo.hs" "haskell" fooContent
-    expectDiagnostics [("Foo.hs", [(DiagnosticSeverity_Error, (3, 6), "Couldn't match type")])]
+    expectDiagnostics [("Foo.hs", [(DiagnosticSeverity_Error, (3, 6), "Couldn't match type", Nothing)])]
```

Comment:
I was expecting at least some of these to now have codes?

Reply:
Yeah, that was a TODO in the PR's opening comment - fixed now in c4d5eddb and 4b452c98